server/services/v1/billing/provider.go

```diff
@@ -0,0 +1,21 @@
+package billing
+
+import (
+	"context"
+
+	"github.com/tigrisdata/tigris/server/config"
+)
+
+type Provider interface {
+	CreateAccount(ctx context.Context, namespaceId string, name string) (string, error)
+	AddDefaultPlan(ctx context.Context, metronomeId string) (bool, error)
+	AddPlan(ctx context.Context, metronomeId string, planId string) (bool, error)
+}
+
+func NewProvider() Provider {
+	if config.DefaultConfig.Billing.Enabled && config.DefaultConfig.Billing.Metronome.Enabled {
```

Comment:
Do we need top-level enabled flag for now?

Reply:
removed